src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+$main-clr: #060b35;
+
+$secondary-clr: #616070;
+
+$accent-blue: #00acdc;
+
+.page__body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: $secondary-clr;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.fw-700 {
+  font-weight: 700;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto 40px;
+}
```

Comment:
Now you are using sass, so in order not to duplicate the class name every time, you can use nesting